pom.xml

```diff
@@ -95,6 +115,22 @@
 			<artifactId>junit-jupiter-engine</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.connectbot</groupId>
+			<artifactId>sshlib</artifactId>
+			<version>2.2.21</version>
+		</dependency>
+		<dependency>
```

Comment:
Remove this dependency 😉.

Reply:
This dependency is required, otherwise the code won't compile